tests/http/test_server.py

```diff
@@ -334,5 +345,101 @@ def test_without_slash_should_redirect(self):
     def test_can_wrap_wsgi_apps(self):
         response = self.fetch("/wsgi/", method="GET")
 
-        assert 200 == response.code
+        assert response.code == 200
         assert "Hello, world!" in tornado.escape.to_unicode(response.body)
+
+
+def default_webapp_factory(config, core):
+    class MainHandler(tornado.web.RequestHandler):
+        def get(self):
+            self.write("Hello from default webapp")
+
+    return [("/", MainHandler, {})]
+
+
+class HttpServerWithAppDefaultWebClient(tornado.testing.AsyncHTTPTestCase):
+    def get_app(self):
+        config = {
+            "http": {
+                "hostname": "127.0.0.1",
+                "port": 6680,
+                "zeroconf": "",
+                "default_webclient": "default_webclient",
+            }
+        }
+        core = mock.Mock()
+
+        apps = [dict(name="default_webclient", factory=default_webapp_factory)]
+
+        http_server = actor.HttpServer(
+            config=config, core=core, sockets=[], apps=apps, statics=[]
+        )
+
+        return tornado.web.Application(http_server._get_request_handlers())
+
+    def test_should_redirect_to_default_webclient(self):
+        response = self.fetch("/", method="GET", follow_redirects=False)
+
+        assert response.code, 302
+        assert response.headers["Location"] == "/default_webclient/"
+
+        response = self.fetch(
+            "/default_webclient/", method="GET", follow_redirects=True
+        )
+
+        assert response.code == 200
+        assert "Hello from default webapp" in tornado.escape.to_unicode(
```

Comment:
I can't find any references to `tornado.escape.to_unicode()` in the Tornado docs, and according to the [implementation](https://github.com/tornadoweb/tornado/blob/master/tornado/escape.py#L219-L229), this is equivalent to just `response.body.decode()`.

Please replace all uses of `tornado.escape.to_unicode(response.body)` in this file with `response.body.decode()`?

Reply:
there were use in others test (than the one I added, so they are updated as well